tests/test_feeds.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+import xml.etree.ElementTree as ET
+
+from django.utils.http import urlquote
+from django.core.urlresolvers import reverse
+
+from . import factories
+
+
+pytestmark = pytest.mark.django_db
+ns = {'feed': 'http://www.w3.org/2005/Atom'}
```

Comment:
yeah that had crossed my mind as I was doing that. I'll change it to `NAMESPACE`.